indy_node/test/test_metadata_functions.py

```diff
@@ -0,0 +1,20 @@
+import pytest
```

Comment:
please check `indy_node/test/package/metadata.py`. It makes sense to merge these tests there